homeassistant/helpers/config_validation.py

```diff
@@ -871,7 +871,7 @@ def script_action(value: Any) -> dict:
     return ACTION_TYPE_SCHEMAS[determine_script_action(value)](value)
 
 
-SCRIPT_SCHEMA = vol.All(ensure_list, [script_action])
+SCRIPT_SCHEMA = vol.All(ensure_list, [vol.Any(script_action, dynamic_template)])
```

Comment:
I don't think that we should add this here. 

In all other places we added this shortcut, it was places where we accepted just conditions. This schema is different in that it accepts actions. We're also going to expand the action schema in the future and I don't know if we can commit to this shortcut here yet.

Reply:
Agree, this PR was made a draft for the same doubts. Furthermore, the frontend doesn't like it either.

Let's mark the shorthand not available within these constructs for now. Especially considering other ideas around future expansion.

At least we have this PR as a future reference.